scripts/zalenium.sh

```diff
@@ -7,6 +7,8 @@ CHROME_CONTAINERS=1
 FIREFOX_CONTAINERS=1
 DESIRED_CONTAINERS=${DESIRED_CONTAINERS:-2}
 MAX_DOCKER_SELENIUM_CONTAINERS=${MAX_DOCKER_SELENIUM_CONTAINERS:-10}
+SELENIUM_CONTAINER_CPU_LIMIT=${SELENIUM_CONTAINER_CPU_LIMIT:100000000}
```

Comment:
I don't think there should be a default, as it should be disabled if not specified.

Reply:
In kubernetes module there are cpuLimit and memLimit values. So I think it should have default value. And when we use without default it would be load server's property so that server could be shutdown. So ı developed like this @pearj

Reply:
It is true that the Kubernetes part has values, but it was developed like that since day 1. The Docker part does not have any default values, and introducing them is not backwards compatible since we don't know what a good default is for all people who are using Zalenium with docker. 

If we arbitrarily set a default value, it is very probable that it will affect a bunch of people and many issues about that will be created, which will increase the maintenance time for the project.

To make it short: yes, technically it makes sense to have defaults, but we are not going to introduce them because this would affect more users than actually benefit them.

Reply:
Ok, @diemol thanks for your review. I will change this request